tasks/task414_mickey_ar_sentence_perturbation_generation.json

```diff
@@ -6,7 +6,11 @@
         "Common Sense Beyond English: Evaluating and Improving Multilingual Language Models for Commonsense Reasoning; Paper Link: https://arxiv.org/pdf/2106.06937.pdf"
     ],
     "Categories": [
-        "Text Modification"
+        "Text Modification -> Paraphrase Generation"
+    ],
+    "Domains": [
+        "Web",
```

Comment:
I guess we decided to remove the web domain.

Reply:
Sorry I have missed this decision. Do you have any suggestions for an alternative domain?

Reply:
Well, not exactly! This dataset is a translation of CSQA and CODAH datasets. The domain of the CSQA is Wikidata, and the domain of CODAH is Commonsense. I think we can use these domains.